.github/workflows/cypress.yml

```diff
@@ -21,6 +21,12 @@ jobs:
           install: true
           # to run component tests we need to use "component: true"
           component: true
+      - name: Upload screenshots
+        uses: actions/upload-artifact@v3
+        if: failure()
+        with:
+          name: component-screenshots
+          path: web/cypress/screenshots
```

Comment:
Do you want to keep this in the PR?
(I think it might be a good idea to do so)

Reply:
Yeah :)

An fyi the artefacts can be found here on the actions summary page:
![image](https://user-images.githubusercontent.com/18622768/227951237-6f842802-6429-459e-b2ac-6c9f0a48fe51.png)

Reply:
Nice! Yeah I think it would be nice to optimize. We will try get automated deployment to production running next week which will free up the staging url for for automated deployment from this repo.